recipe/meta.yaml

```
@@ -15,13 +15,14 @@ source:
 build:
     number: 1004
```

```suggestion
    number: 1005
```

Please bump the build number to differentiate the builds